tests/positive/security/tls.go

```diff
@@ -89,9 +67,22 @@ AKbyaAqbChEy9CvDgyv6qxTYU+eeBImLKS3PH2uW5etc/69V/sDojqpH3hEffsOt
 			}
 		}`)
 
+	customCAServerFile2 = []byte(`{
+			"ignition": { "version": "3.0.0" },
+			"storage": {
+				"files": [{
+					"path": "/foo/bar2",
+					"contents": { "source": "data:,example%20file1%0A" }
```

Comment:
s/file1/file2/, not that it really matters here.

Reply:
Fixed